src/uct/api/uct.h

```diff
@@ -419,6 +419,10 @@ typedef enum uct_atomic_op {
                                                        channel with remote peer is broken, even if there
                                                        are no outstanding send operations */
 
+        /* Active Message alignment */
+#define UCT_IFACE_FLAG_AM_ALIGNMENT   UCS_BIT(47) /**< Interface supports setting a specific alignment
```

Comment:
how would this help if we can check it after already created the interface?
maybe iface should return the existing alignment guarantee?

Reply:
we will have to distinguish what lane to pick if user requested specific alignment during worker creation.

> maybe iface should return the existing alignment guarantee?

Can you please elaborate?